python/cairo-ec/src/cairo_ec/ec_ops.cairo

```diff
@@ -58,11 +68,29 @@ func get_random_point{
     return get_random_point(seed=seed, a=a, b=b, g=g, p=p);
 }
 
-// Add two EC points. Doesn't check if the inputs are on curve nor if they are the point at infinity.
+// / @notice Adds two EC points on the ALT_BN128 curve.
+// / @dev Handles the point at infinity (0, 0) by returning the other point if either input is infinity.
+// /      For points with the same x-coordinate modulo modulus, returns infinity if they are inverses (p.y + q.y = 0 mod P),
+// /      or doubles the point if they are equal. Otherwise, performs standard addition for distinct points.
+// /      Does not check if inputs lie on the curve; this is the caller's responsibility.
+// / @param p The first elliptic curve point as a G1Point struct.
+// / @param q The second elliptic curve point as a G1Point struct.
+// / @param a The curve coefficient 'a' as a UInt384.
+// / @param modulus The prime modulus of the field as a UInt384.
+// / @return The resulting point from p + q on the ALT_BN128 curve as a G1Point.
 func ec_add{range_check96_ptr: felt*, add_mod_ptr: ModBuiltin*, mul_mod_ptr: ModBuiltin*}(
```

Comment:
this should be ec_add_unchecked then
ec_add_unchecked, which is the compiled circuit, should be named differently imo